Code/autopkglib/__init__.py

```diff
@@ -216,11 +216,11 @@ def _get_macos_prefs(self):
 
     def _get_file_prefs(self):
         r"""Lookup preferences for Windows in a standardized path, such as:
-        * `C:\\Users\username\AppData\Local\Autopkg\config.{plist,json}`
+        * `C:\Users\username\AppData\Roaming\Autopkg\config.{plist,json}`
         * `/home/username/.config/Autopkg/config.{plist,json}`
         Tries to find `config.plist`, then `config.json`."""
-
-        config_dir = appdirs.user_config_dir(APP_NAME, appauthor=False)
+        # Windows configuration files should go to the Appdata\Roaming dir.
+        config_dir = appdirs.user_config_dir(APP_NAME, appauthor=False, roaming=True)
```

Comment:
I wonder if it should be checking roaming first, but then check local as a fallback?

I think this change would break my autopkg prefs on my windows devices until I moved the prefs file from local to roaming.